assets/js/services/global-vars.js

```diff
@@ -3,6 +3,24 @@ export const COLLECTION_TYPES = [
   { label: "NUL Theme", id: "NUL Theme", value: "NUL Theme" },
 ];
 
+export const SORT_OPTIONS = [
+  {
+    sortBy: "desc",
+    dataField: "_score",
+    label: "Sort By Relevancy",
+  },
+  {
+    sortBy: "asc",
+    dataField: "modifiedDate",
+    label: "Sort By Modified Date",
+  },
+  {
+    sortBy: "asc",
+    dataField: "descriptiveMetadata.title.keyword",
+    label: "Sort By Title",
+  },
+];
+
```

Comment:
@divyareddynu I know it's small, but can we make this variable constant title a little more specific?  How about something like `REACTIVESEARCH_SORT_OPTIONS`?   Think of doing quick scans like in 6 months of the code, and what will help us quickly understand it without even looking at the code.   Everything else here looks great.